core/vm/logger_json.go

```diff
@@ -87,3 +88,28 @@ func (l *JSONLogger) CaptureEnd(output []byte, gasUsed uint64, t time.Duration,
 	}
 	l.encoder.Encode(endLog{common.Bytes2Hex(output), math.HexOrDecimal64(gasUsed), t, errMsg})
 }
+
+func (l *JSONLogger) CaptureEnter(typ OpCode, from common.Address, to common.Address, input []byte, gas uint64, value *big.Int) {
+	frame := StructFrame{
+		Type:  typ.String(),
+		From:  from,
+		To:    to,
+		Input: input,
+		Gas:   gas,
+		Value: value,
+	}
+	l.encoder.Encode(frame)
```

Comment:
Hm, I'm not sure we should add this here. The JSONLogger is used for opcode tracing, in `standardPrintBlockToFile` and evm fuzzing, and we don't want to mix and match opcode output with scope output. So maybe just remoe this for now.. 
```suggestion
```

Reply:
Yeah, we don't want this: 
```
[user@work evm]$ go run . --json --code 6040604060406040604060406040ff run  | tail -n4
{"pc":14,"op":255,"gas":"0x2540be3eb","gasCost":"0x1db0","memory":"0x","memSize":0,"stack":["0x40","0x40","0x40","0x40","0x40","0x40","0x40"],"returnData":"0x","depth":1,"refund":0,"opName":"SELFDESTRUCT","error":""}
{"type":"SELFDESTRUCT","from":"0x0000000000000000000000007265636569766572","to":"0x0000000000000000000000000000000000000040","input":"0x","gas":"0x0","value":"0x0","error":""}
{"output":"0x","gasUsed":"0x0"}
{"output":"","gasUsed":"0x1dc5","time":173146}
```

Reply:
I was using it as in your example for testing. But I can see why we wouldn't want it. Removed